test/unit/lib/tasks/creation_task_definition_test.rb

```diff
@@ -0,0 +1,57 @@
+require 'test_helper'
+
+class CreationTaskDefinitionTest < ActiveSupport::TestCase
+  let(:definition_class) { ForemanDeployments::Tasks::CreationTaskDefinition }
+
+  describe 'build' do
+    test 'it returns instance of HostCreationTaskDefinition for Host' do
+      definition = definition_class.build('class' => 'Host')
+      assert_equal(ForemanDeployments::Tasks::HostCreationTaskDefinition, definition.class)
+    end
+
+    test 'it returns instance of HostCreationTaskDefinition for Host::Managed' do
+      definition = definition_class.build('class' => 'Host::Managed')
+      assert_equal(ForemanDeployments::Tasks::HostCreationTaskDefinition, definition.class)
+    end
+
+    test 'it returns instance of HostCreationTaskDefinition for other classes' do
```

Comment:
Copy - paste typo: "it returns instance of **HostCreationTaskDefinition** for..." should be "CreationTaskDefinition"